test/service/Cargo.toml

```diff
@@ -47,7 +47,7 @@ sp-timestamp = { git = "https://github.com/paritytech/substrate", branch = "mast
 sp-trie = { git = "https://github.com/paritytech/substrate", branch = "master" }
 substrate-test-client = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sc-cli = { git = "https://github.com/paritytech/substrate", branch = "master" }
-clap = { version = "3.2", features = ["derive"] }
+clap = { version = "3.2.5", features = ["derive", "deprecated"] }
```

Comment:
Why do we need `deprecated`?

Reply:
> Also added the "deprecated" feature to clap so no cli options will be added in the old style anymore.

Now I read this.

But what does that mean?

Reply:
So clap deprecated some of their derive API to streamline their API for the upcoming 4.0 release.  However, the deprecation messaged did not make it obvious how to migrate. So after user feedback in https://github.com/clap-rs/clap/issues/3822, they hid the deprecation messages behind feature flag "deprecated".

However, since I have already adopted the new API now, we want the deprecation messages for the old style so no one will go back.